scripts/postgres_resetdb.sh

```diff
@@ -63,6 +68,16 @@ main() {
         die "Error: Failed to create database '${POSTGRES_DB}'."
       psql "${FLAGS[@]}" -d ${POSTGRES_DB} -f ${TRILLIAN_PATH}/storage/postgres/storage.sql || \
         die "Error: Failed to create tables in '${POSTGRES_DB}' database."
+      if ! psql "${FLAGS[@]}" -t -c "SELECT 1 FROM pg_user WHERE usename = '${POSTGRES_USER}'" | grep -q 1; then
+        psql "${FLAGS[@]}" -c "CREATE USER ${POSTGRES_USER} WITH PASSWORD '${POSTGRES_PASSWORD}';" || \
```

Comment:
You can add `WITH SUPERUSER` to the `CREATE USER` query to avoid having to use `ALTER ROLE` later.